### PR TITLE
Fold long lines to max-line-length=88

### PR DESCRIPTION
**Pull request by iandrc**
I've inserted some `# noqa: E501` to ignore some lines. Otherwise, the folding may harm readability.

There are still some unfolded lines, and I'll fix them but I need a bit more info first. Namely, I have to deal with the `./pylib/gyp/generator/make.py` file. Some of the lines are inside `""" """`. Therefore, it's not obvious what's the best option. Please, advise :)

Refs: https://github.com/nodejs/gyp-next/issues/56

### CODE REVIEW

File: pylib/gyp/MSVSSettings_test.py [hunk 1]
```diff
@@ -678,7 +678,8 @@ def testConvertToMSBuildSettings_warnings(self):
                 "MSBuild, index value (21) not in expected range [0, 3)",
                 "Warning: while converting VCCLCompilerTool/UsePrecompiledHeader to "
                 "MSBuild, index value (13) not in expected range [0, 3)",
-                "Warning: while converting VCCLCompilerTool/GeneratePreprocessedFile to "
+                "Warning: while converting "
```
Comment (iandrc) [inline pylib/gyp/MSVSSettings_test.py:681]: I'm not sure it's OK

File: pylib/gyp/MSVSSettings_test.py [hunk 2]
```diff
@@ -1348,7 +1349,7 @@ def testConvertToMSBuildSettings_actual(self):
                 "EmbedManifest": "false",
                 "GenerateCatalogFiles": "true",
                 "InputResourceManifests": "asfsfdafs",
-                "ManifestResourceFile": "$(IntDir)\\$(TargetFileName).embed.manifest.resfdsf",
+                "ManifestResourceFile": "$(IntDir)\\$(TargetFileName).embed.manifest.resfdsf",  # noqa: E501
```
Comment (iandrc) [inline pylib/gyp/MSVSSettings_test.py:1352]: I've inserted `# noqa: 501` because doing otherwise may impair readability

Comment (cclauss) [inline pylib/gyp/MSVSSettings_test.py:1352]:
```suggestion
                "ManifestResourceFile":
                    "$(IntDir)\\$(TargetFileName).embed.manifest.resfdsf",
```

File: pylib/gyp/MSVSSettings_test.py [hunk 3]
```diff
@@ -1532,7 +1533,7 @@ def testConvertToMSBuildSettings_actual(self):
                 "LinkIncremental": "",
             },
             "ManifestResourceCompile": {
-                "ResourceOutputFileName": "$(IntDir)$(TargetFileName).embed.manifest.resfdsf"
+                "ResourceOutputFileName": "$(IntDir)$(TargetFileName).embed.manifest.resfdsf"   # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/MSVSSettings_test.py:1536]:
```suggestion
                "ResourceOutputFileName":
                    "$(IntDir)$(TargetFileName).embed.manifest.resfdsf"
```

File: pylib/gyp/generator/make.py [hunk 4]
```diff
@@ -1342,7 +1342,7 @@ def WriteSources(
             )
 
             if self.flavor == "mac":
-                cflags = self.xcode_settings.GetCflags(configname, arch=config.get('xcode_configuration_platform'))
+                cflags = self.xcode_settings.GetCflags(configname, arch=config.get('xcode_configuration_platform'))   # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:1345]:
```suggestion
                cflags = self.xcode_settings.GetCflags(
                    configname,
                    arch=config.get('xcode_configuration_platform')
                )
```

File: pylib/gyp/generator/ninja.py [hunk 5]
```diff
@@ -1513,7 +1515,7 @@ def WriteLinkForArch(
             if self.flavor != "win":
                 link_file_list = output
                 if self.is_mac_bundle:
-                    # 'Dependency Framework.framework/Versions/A/Dependency Framework' ->
+                    # 'Dependency Framework.framework/Versions/A/Dependency Framework' ->   # noqa: E501
                     # 'Dependency Framework.framework.rsp'
```
Comment (cclauss) [inline pylib/gyp/generator/ninja.py:1519]:
```suggestion
                    # 'Dependency Framework.framework/Versions/A/Dependency Framework'
                    # -> 'Dependency Framework.framework.rsp'
```

File: pylib/gyp/generator/ninja.py [hunk 6]
```diff
@@ -2535,9 +2538,9 @@ def GenerateOutputForConfig(target_list, target_dicts, data, params, config_name
             "solink",
             description="SOLINK $lib",
             restat=True,
-            command=mtime_preserving_solink_base % {"suffix": "@$link_file_list"},
+            command=mtime_preserving_solink_base % {"suffix": "@$link_file_list"},  # noqa: E501
             rspfile="$link_file_list",
-            rspfile_content="-Wl,--whole-archive $in $solibs -Wl,--no-whole-archive $libs",
+            rspfile_content="-Wl,--whole-archive $in $solibs -Wl,--no-whole-archive $libs",  # noqa: E501, E261
```
Comment (cclauss) [inline pylib/gyp/generator/ninja.py:2543]:
```suggestion
            rspfile_content=("-Wl,--whole-archive $in $solibs -Wl,"
                             "--no-whole-archive $libs"),
```

File: pylib/gyp/generator/xcode.py [hunk 7]
```diff
@@ -735,7 +735,7 @@ def GenerateOutput(target_list, target_dicts, data, params):
             "loadable_module+xcuitest": "com.apple.product-type.bundle.ui-testing",
             "shared_library+bundle": "com.apple.product-type.framework",
             "executable+extension+bundle": "com.apple.product-type.app-extension",
-            "executable+watch+extension+bundle": "com.apple.product-type.watchkit-extension",
+            "executable+watch+extension+bundle": "com.apple.product-type.watchkit-extension",   # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/xcode.py:738]:
```suggestion
            "executable+watch+extension+bundle":
                "com.apple.product-type.watchkit-extension",
```

**Comment by iandrc** [comment]
@cclauss thank you! You really helped me.

I've added some more changes, please have a look when you have spare time :)

File: pylib/gyp/generator/make.py [hunk 8]
```diff
@@ -177,7 +177,7 @@ def CalculateGeneratorInputInfo(params):
 
 quiet_cmd_solink_module = SOLINK_MODULE($(TOOLSET)) $@
 cmd_solink_module = $(LINK.$(TOOLSET)) -o $@ -shared $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -Wl,-soname=$(@F) -Wl,--start-group $(filter-out FORCE_DO_CMD, $^) -Wl,--end-group $(LIBS)
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:180]:
```suggestion
"""  # noqa: E501
```
`<space><space>#` please

File: pylib/gyp/generator/make.py [hunk 9]
```diff
@@ -191,7 +191,7 @@ def CalculateGeneratorInputInfo(params):
 
 quiet_cmd_solink_module = SOLINK_MODULE($(TOOLSET)) $@
 cmd_solink_module = $(LINK.$(TOOLSET)) -bundle $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -o $@ $(filter-out FORCE_DO_CMD, $^) $(LIBS)
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:194]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/make.py [hunk 10]
```diff
@@ -218,7 +218,7 @@ def CalculateGeneratorInputInfo(params):
 cmd_solink_module = $(LINK.$(TOOLSET)) -shared $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -Wl,-soname=$(@F) -o $@ -Wl,--start-group $(filter-out FORCE_DO_CMD, $^) -Wl,--end-group $(LIBS)
 quiet_cmd_solink_module_host = SOLINK_MODULE($(TOOLSET)) $@
 cmd_solink_module_host = $(LINK.$(TOOLSET)) -shared $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -Wl,-soname=$(@F) -o $@ $(filter-out FORCE_DO_CMD, $^) $(LIBS)
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:221]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/make.py [hunk 11]
```diff
@@ -236,7 +236,7 @@ def CalculateGeneratorInputInfo(params):
 
 quiet_cmd_solink_module = SOLINK_MODULE($(TOOLSET)) $@
 cmd_solink_module = $(LINK.$(TOOLSET)) -shared $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -o $@ $(filter-out FORCE_DO_CMD, $^) $(LIBS)
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:239]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/make.py [hunk 12]
```diff
@@ -254,8 +254,7 @@ def CalculateGeneratorInputInfo(params):
 
 quiet_cmd_solink_module = SOLINK_MODULE($(TOOLSET)) $@
 cmd_solink_module = $(LINK.$(TOOLSET)) $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -o $@ $(filter-out FORCE_DO_CMD, $^) $(LIBS) -Wl,DLL
-
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:257]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/make.py [hunk 13]
```diff
@@ -404,7 +403,7 @@ def CalculateGeneratorInputInfo(params):
 cmd_copy = ln -f "$<" "$@" 2>/dev/null || (rm -rf "$@" && cp %(copy_archive_args)s "$<" "$@")
 
 %(link_commands)s
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:406]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/make.py [hunk 14]
```diff
@@ -503,7 +502,7 @@ def CalculateGeneratorInputInfo(params):
 .PHONY: FORCE_DO_CMD
 FORCE_DO_CMD:
 
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:505]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/make.py [hunk 15]
```diff
@@ -534,7 +533,7 @@ def CalculateGeneratorInputInfo(params):
 
 quiet_cmd_infoplist = INFOPLIST $@
 cmd_infoplist = $(CC.$(TOOLSET)) -E -P -Wno-trigraphs -x c $(INFOPLIST_DEFINES) "$<" -o "$@"
-"""
+"""     # noqa: E501
```
Comment (cclauss) [inline pylib/gyp/generator/make.py:536]:
```suggestion
"""  # noqa: E501
```

File: pylib/gyp/generator/msvs.py [hunk 16]
```diff
@@ -753,7 +754,7 @@ def _EscapeEnvironmentVariableExpansion(s):
 
   Returns:
       The escaped string.
-  """
+  """   # noqa: E731,E123,E501
```
Comment (cclauss) [inline pylib/gyp/generator/msvs.py:757]:
```suggestion
  """  # noqa: E731,E123,E501
```

Comment (cclauss) [reply pylib/gyp/generator/msvs.py:757]: Why all this?

**Comment by iandrc** [comment]
@cclauss thanks again for the review

I apologize for all the space-related issues. Didn't notice it at first

**Comment by iandrc** [comment]
@cclauss Is everything OK with this PR? Should I redo something?

**Comment by cclauss** [comment]
You already have a green review from me.  Let's allow the others to review as well.

**Pull request merged**